Octokit.Reactive/Clients/IObservableIssuesEventsClient.cs

```diff
@@ -14,8 +20,17 @@ public interface IObservableIssuesEventsClient
         /// <param name="owner">The owner of the repository</param>
         /// <param name="name">The name of the repository</param>
         /// <param name="number">The issue number</param>
-        /// <returns></returns>
         IObservable<EventInfo> GetAllForIssue(string owner, string name, int number);
+        
+        /// <summary>
+        /// Gets all events for the issue.
+        /// </summary>
+        /// <remarks>
+        /// http://developer.github.com/v3/issues/events/#list-events-for-an-issue
+        /// </remarks>
+        /// <param name="repositoryId">The ID of the repository</param>
```

Comment:
We've always followed the [.NET Framework Design Guidelines for capitalization](https://msdn.microsoft.com/en-us/library/ms229043(v=vs.100).aspx). 

> The two abbreviations that can be used in identifiers are ID and OK. __In Pascal-cased identifiers they should appear as `Id`, and `Ok`__. If used as the first word in a camel-cased identifier, they should appear as id and ok, respectively.

So I think it should be `Id`.